netlify.toml

```diff
@@ -1076,4 +1081,14 @@ status = 301
 [[redirects]]
   from = "/account/my-organization/team-activity-log"
   to = "/account/my-organization/profile-and-team/team-activity-log"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/sso-providers-configuration/sso-login/single-sign-on-login"
+  to = "/account/my-organization/integrations/authentications/sso-providers-configuration/sso-login/single-sign-on-login"
+  status = 301
+
+[[redirects]]
+  from = "/account/my-organization/sso-providers-configuration/sso-login/okta-saml"
```

Comment:
As I understand it, the following URL is no longer used:

> "/account/my-organization/sso-providers-configuration/sso-login/okta-saml"

So line 405 should also be updated with the new URL.